lib/active_admin/views/components/paginated_collection.rb

```diff
@@ -41,6 +41,7 @@ def build(collection, options = {})
         @params         = options.delete(:params)
         @param_name     = options.delete(:param_name)
         @download_links = options.delete(:download_links)
+        @download_url_options = options.delete(:download_url_options) || {}
```

Comment:
change this to:
@download_url_options = options.delete(:download_url_options, {})